Makefile

```diff
@@ -47,6 +47,15 @@ TEST_DAMENG_HOST ?= dameng:5236
 TEST_DAMENG_USERNAME ?= SYSDBA
 TEST_DAMENG_PASSWORD ?= SYSDBA
 
+TEST_YDB_SCHEME ?= grpc
```

Comment:
Maybe single parameter like `TEST_YDB_CONNECTION_STRING` instead 8 parameters?

Reply:
`TEST_YDB_CONNECTION_STRING=grpc://localhost:2136/local?qo_query_bind=table_path_prefix(/local/xorm/test),declare,numeric&go_fake_tx=scan,scheme,scripting`
or
`TEST_YDB_DSN=grpc://localhost:2136/local?qo_query_bind=table_path_prefix(/local/xorm/test),declare,numeric&go_fake_tx=scan,scheme,scripting`

Reply:
I will combine these parameters into one:
```
TEST_YDB_DBNAME ?= local
TEST_YDB_TABLE_PATH_PREFIX ?= /local/xorm/test
TEST_YDB_QUERY_BIND ?= table_path_prefix($(TEST_YDB_TABLE_PATH_PREFIX)),declare,numeric
TEST_YDB_FAKE_TX ?= scan,scheme,scripting
```
But will keep `scheme` and `host` for secure connection test
```
TEST_YDB_SCHEME ?= grpc
TEST_YDB_HOST ?= ydb:2136
```

Reply:
for secure connection you can change connection string to
```
TEST_YDB_CONNECTION_STRING=grpcs://localhost:2135/local?qo_query_bind=table_path_prefix(/local/xorm/test),declare,numeric&go_fake_tx=scan,scheme,scripting
```

Reply:
You have only one usage of envs TEST_YDB_SCHEME && TEST_YDB_HOST
I think you dont make difficult if you can make easy instead